pkgs/by-name/sc/scopehal-apps/package.nix

```diff
@@ -0,0 +1,102 @@
+{
+  stdenv,
+  breakpointHook,
+  fetchFromGitHub,
+  cmake,
+  git,
+  lib,
+  pkg-config,
+  libsigcxx,
+  gtkmm3,
+  cairomm,
+  yaml-cpp,
+  catch2,
+  glfw,
+  libtirpc,
+  liblxi,
+  glew,
+  libllvm,
+  libdrm,
+  elfutils,
+  libxcb,
+  zstd,
+  libxshmfence,
+  wrapGAppsNoGuiHook,
+  xcbutilkeysyms,
+  systemd,
+  shaderc,
+  vulkan-headers,
+  vulkan-loader,
+  vulkan-tools,
+  glslang,
+  spirv-tools,
+  ffts,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "scopehal-apps";
+  version = "e75c0bd1eed4c057bfbbfa6a75a211a88d30c934";
+
+  src = fetchFromGitHub {
+    owner = "ngscopeclient";
+    repo = "scopehal-apps";
+    rev = version;
+    hash = "sha256-sKrxElL4XRHqCPHsH8Q/2pJeQwnMex0YAQMU5I4FEVA=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    git
+    wrapGAppsNoGuiHook
+  ];
+
+  buildInputs = [
+    pkg-config
+    libsigcxx
+    gtkmm3
+    cairomm
+    yaml-cpp
+    catch2
+    glfw
+    libtirpc
+    liblxi
+    glew
+    libllvm
+    libdrm
+    elfutils
+    libxcb
+    zstd
+    libxshmfence
+    xcbutilkeysyms
+    systemd
+    vulkan-headers
+    vulkan-loader
+    vulkan-tools
+    spirv-tools
+    glslang
+    shaderc
+    ffts
+  ];
+
+  enableParallelBuilding = true;
+
+  # Targets InitializeSearchPaths
+  postPatch = ''
+    substituteInPlace lib/scopehal/scopehal.cpp \
+      --replace '"/share/' '"/../share/'
+  '';
+
+  cmakeFlags = [ "-DCURRENT_GIT_VERSION=${lib.substring 0 7 src.rev}" ];
+
+  meta = with lib; {
+    homepage = "https://www.ngscopeclient.org/";
+    license = licenses.bsd3;
+    maintainers = with lib.maintainers; [
+      thoughtpolice
+      hansfbaier
```

Comment:
Same question as for ffts package: does `thoughtpolice` and `hansfbaier` want to maintain this package?

Reply:
I put in @thoughtpolice, because seems to be everywhere.
I am not sure if I have the necessary experience in maintaining this.
But I basically might volunteer, but you seem to have much more flight time with doing this.